apps/wallet/src/ui/app/hooks/useGetObject.ts

```diff
@@ -6,7 +6,7 @@ import { useQuery, type UseQueryResult } from '@tanstack/react-query';
 
 import { useRpc } from '_hooks';
 
-export function useGetObjectData(
+export function useGetObject(
     objectId: string
```

Comment:
Something I just thought of here: We should normalize the object ID when we fetch it so that we can have the most cache hits possible (e.g. `0x2` and `0x0000000000000000000000000000000000000002` should be cached under the same entry).

We can do this later though, added https://mysten.atlassian.net/browse/APPS-299 to track for now.

Reply:
done